packages/shared/src/components/plus/PlusTrustRefund.tsx

```diff
@@ -2,54 +2,36 @@ import type { ComponentProps, ReactElement } from 'react';
 import React from 'react';
 import classNames from 'classnames';
 import { IconSize } from '../Icon';
-import { BellIcon, PrivacyIcon, TipIcon } from '../icons';
+import { PrivacyIcon } from '../icons';
 import {
   Typography,
   TypographyColor,
   TypographyType,
 } from '../typography/Typography';
-import { useViewSize, ViewSize } from '../../hooks';
-import { usePaymentContext } from '../../contexts/payment/context';
 
 type PlusTrustRefundProps = ComponentProps<'div'>;
 
 const alerts = {
-  trial: {
-    icon: TipIcon,
-    iconClassName: 'bg-action-upvote-float text-action-upvote-default',
-    text: '7-day free trial, Cancel anytime.',
-  },
   refund: {
     icon: PrivacyIcon,
     iconClassName: 'bg-action-comment-float text-accent-blueCheese-default',
     text: '30 day hassle-free refund. No questions asked.',
   },
-  reminder: {
-    icon: BellIcon,
-    iconClassName: 'bg-action-bookmark-float text-accent-bun-default',
-    text: "We'll email you a reminder before your trial ends.",
-  },
 };
 
 export const PlusTrustRefund = ({
   className,
   ...attrs
 }: PlusTrustRefundProps): ReactElement => {
-  const isLaptop = useViewSize(ViewSize.Laptop);
-  const { isFreeTrialExperiment } = usePaymentContext();
-  const hasVerticalLayout = isFreeTrialExperiment && isLaptop;
-  const items = isFreeTrialExperiment
-    ? [alerts.trial, alerts.refund, alerts.reminder]
-    : [alerts.refund];
+  const items = [alerts.refund];
 
   return (
     <>
       {items.map(({ icon: Icon, iconClassName, text }) => (
```

Comment:
Does this have to be an array and map when it's just one item? 🙈

Reply:
Right. Yeah, we can now just render this alone. Didn't realize that 😂